addons/sale/models/sale_order_line.py

```diff
@@ -320,7 +328,7 @@ def _compute_name(self):
                 elif dp_state == 'cancel':
                     name = _("%(line_description)s (Canceled)", line_description=name)
```

Comment:
`name` given as `line_description` is now undefined.
I suppose it's not tested since the runbot is green, but it'll probably raise in existing flows.